terraform/aws/scenarios/omnibus-restore-backup/files/chef-server.rb

```diff
@@ -0,0 +1,11 @@
+opscode_erchef['keygen_start_size'] = 30
+
+opscode_erchef['keygen_cache_size']=60
+
+nginx['ssl_dhparam']='/etc/opscode/dhparam.pem'
+
+insecure_addon_compat false
+
+data_collector['token'] = 'foobar' unless data_collector.nil?
```

Comment:
Copied this but added the `unless` clauses to ensure this config works with older server versions. Want me to copy this to the other `chef-server.rb` files in the other scenarios?

Reply:
That would be awesome!

Reply:
Added in 8ed385c00